src/sass/rx/_partials/_rx-prescription-card.scss

```diff
@@ -1,4 +1,4 @@
-/* Prescription cards for Activity page. */
+/* Prescription cards for Active page. */
```

Comment:
Did you mean to remove this file and make this change on `_rx-prescriptions.scss` instead?